src/main/java/com/jackbaretto/scrumtest/extractor/MCQExtractor.java

```diff
@@ -46,16 +45,16 @@ private ExtractionResult recognizeCharacters(final File mcqPicture) {
     /**
      * Instanciates the OCR
      */
-    private static ITesseract createOcr() {
+    private ITesseract createOcr() {
 
         //JNA Interface Mapping
-        final ITesseract ocr = new Tesseract();
+        final ITesseract tesseract = new Tesseract();
```

Comment:
I agree with you. I change the name because "ocr" which hides the field "ocr". I will find a better name